cmd/mo-service/config.go

```diff
@@ -37,7 +36,9 @@ import (
 	"github.com/matrixorigin/matrixone/pkg/logutil"
 	"github.com/matrixorigin/matrixone/pkg/pb/metadata"
 	"github.com/matrixorigin/matrixone/pkg/perfcounter"
+	"github.com/matrixorigin/matrixone/pkg/util/metric/stats"
 	tomlutil "github.com/matrixorigin/matrixone/pkg/util/toml"
+	_version "github.com/matrixorigin/matrixone/pkg/version"
```

Comment:
Do not use underscores at the beginning of package names

Reply:
but “version” is a conflicting ident